src/controllers/sharedExecutionPlanUtils.ts

```diff
@@ -107,6 +107,15 @@ export async function createExecutionPlanGraphs(
             // malformed xml
```

Comment:
Maybe worth elaborating on this comment to say something like `// display error at first instance of malformed XML